code/AssetLib/glTF2/glTF2Asset.inl

```diff
@@ -400,7 +401,10 @@ inline void Buffer::Read(Value &obj, Asset &r) {
         }
     } else { // Local file
         if (byteLength > 0) {
-            std::string dir = !r.mCurrentAssetDir.empty() ? (r.mCurrentAssetDir) : "";
+            std::string dir = !r.mCurrentAssetDir.empty() ? (
```

Comment:
I need to improve this. When using zwo similar implementations for the same job I need to refactor.